src/MainCase/MainCaseLid.scad

```diff
@@ -58,20 +59,20 @@ module MainCaseLid() {
       translate([36, 39, 0])
       rotate([0, 0, 90])
       BatteryHolder();
+      minkowski(){
```

Comment:
minkowski allows for an easy chamfer without causing too much complexity when using a pyramid. 
![image](https://user-images.githubusercontent.com/44007906/148275302-abe6be0c-ea23-43c1-a5d1-427385f90045.png)

Reply:
Yeah, but it is super slow :( I had a minkowski there earlier and removed it because I was unhappy with it.

Reply:
A test run without this minkowski and flushed caches for me takes 24 seconds to build from scratch.
A test run with this minkowski and flushed caches takes about 55 seconds. 

I suggest we handle this by only using time-intensive the chamfer pyramid in non-preview mode - during stl build I don't think the compute time is too much to ask. I'll add that change.

I also explored using the trick with a shifted, smaller polygon and hull, but because this shape has a hole in it that trick isn't as easily applied. It would be possible by designing an (in the mathematical sense) star-shaped cutter to cut the hole, but Imho the toll taken in readability would not justify the gains of a factor two in build time.